data/tools.yaml

```diff
@@ -342,3 +342,6 @@
   url: https://github.com/KittyHeaven/blue_bird
   tags:
     - testing
+- name: esplanade
+  summary: Validate requests and responses against API Blueprint specifications.
```

Comment:
This seems to be Ruby specific, if that is correct could you please mention it is for Ruby in the summary so its clear to people searching down list.

Reply:
That's right, it's better this way.